docs/dev/conventions/naming_conventions.md

```diff
@@ -1,11 +1,29 @@
-Casing Types:
-- [Kebab-case](https://en.wikipedia.org/wiki/Kebab_case)
-
 ## Java
 
-- In general, use the naming conventions from the [Google Java Style Guide](https://google.github.io/styleguide/javaguide.html#s5-naming).
-- Use a `new` prefix for factory methods (e.g. `newFoo()` instead of `createFoo()`).
+- Follow the naming in: [Google Java Style Guide](https://google.github.io/styleguide/javaguide.html#s5-naming)
+- Use a `new` prefix for factory methods (e.g. `newFoo()` instead of `createFoo()`)
 
 ## URLs
 
-- kebab-case
+- [kebab-case](https://en.wikipedia.org/wiki/Kebab_case)
+
+
+## Database
+
+- [snake_case](https://en.wikipedia.org/wiki/Snake_case) 
+- Use singular names, eg: `user` instead of `users`
+- Use `id` for primary key column name
```

Comment:
I think I get what you mean, but I think this short sentence is too vague:
Currently the DB uses varchars an primary key, which is the username. This column should be renamed to id, but an id column (with an int value) should be introduced instead.

Reply:
Agree on introducing a column. The important point is the name of the introduced column, I add d7d274dd5 to give an example and add a mention to 'favor' numeric PK column.